docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -46,21 +45,21 @@ User=root
 ----
 
 Save the file and reload systemd with `systemctl daemon-reload` and restart OpenNMS with `systemctl restart opennms`.
-The OpenNMS process should run as root user instead of `opennms` and can be verified with `ps -aux | grep java`.
+The OpenNMS process should run as the `root` user instead of `opennms` and can be verified with `ps -aux | grep java`.
```

Comment:
```suggestion
The {page-component-title} process should run as the `root` user instead of `opennms` and can be verified with `ps -aux | grep java`.
```